libs/qa/common_df_diff.py

```diff
@@ -82,8 +82,8 @@ def __str__(self):
 TS only in this file: {self.my_ts}
 TS points only in this file: {self.my_ts_points.groupby('date').size().to_dict()}
 """ + (
-            f"""TS diffs:\n{self.ts_diffs.sort_values('diff', ascending=False)}
-TS diffs by variable and has_overlap:\n{self.ts_diffs.groupby('variable has_overlap'.split()).mean()}
+            f"""TS diffs:\n{self.ts_diffs.sort_values('diff', ascending=False).head(20)}
+TS diffs by variable and has_overlap:\n{self.ts_diffs.groupby('variable has_overlap'.split()).mean().sort_values('diff', ascending=False).head(20)}
```

Comment:
Seems like it might be worth pulling this calculation out from the template string and into a variable